docs/development/contributing_guide.md

```diff
@@ -48,19 +48,27 @@ can find many good git tutorials on the web.
 
 # 4. Install the dependencies
 
-Once you have installed Python 3 and added the source, please open a terminal and
-setup a *virtualenv*, as follows:
+Synapse uses the [poetry](https://python-poetry.org/) project to manage its dependencies
+and development environment. Once you have installed Python 3 and added the
+source, you should install `poetry`. We recommend
```

Comment:
It seems that there is a typo here? "We recommend" is followed by "Of their installation methods..."

Reply:
Thanks!

```suggestion
source, you should install `poetry`.
```